### PR TITLE
[ALLUXIO-2250] Add Mesos doc

### PR DESCRIPTION
**Pull request by aaudiber**
https://alluxio.atlassian.net/browse/ALLUXIO-2250

image

### CODE REVIEW

File: docs/_data/table/cluster-management.csv [hunk 1]
```diff
@@ -1,16 +1,16 @@
 propertyName,defaultValue
 alluxio.integration.master.resource.cpu,1
 alluxio.integration.master.resource.mem,1024 MB
-alluxio.integration.mesos.executor.dependency.path,https://s3.amazonaws.com/alluxio-mesos
-alluxio.integration.mesos.jre.path,jre1.7.0_79
-alluxio.integration.mesos.jre.url,https://s3.amazonaws.com/alluxio-mesos/jre-7u79-linux-x64.tar.gz
+alluxio.integration.mesos.executor.dependency.path,http://downloads.alluxio.org/downloads/files/${alluxio.version}/alluxio-${alluxio.version}-bin.tar.gz
+alluxio.integration.mesos.jdk.path,jdk1.7.0_79
+alluxio.integration.mesos.jdk.url,https://s3-us-west-2.amazonaws.com/alluxio-mesos/jdk-7u79-linux-x64.tar.gz
```
Comment (gpang) [inline docs/_data/table/cluster-management.csv]: would something like `https://alluxio-mesos.s3.amazonaws.com/jdk-7u79-linux-x64.tar.gz` work? I'm wondering if could get rid of the `us-west-2` part.

File: docs/en/Running-Alluxio-on-Mesos.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+1. Install Alluxio on all Mesos nodes.
+2. Point Mesos to an Alluxio tarball.
+
+#### Setting configuration properties
+See the [Configuration Settings](Configuration-Settings.html) documentation for instructions on how to set configuration properties.
+
+#### Deploy with Alluxio already installed on all Mesos nodes
+
+1. Install Alluxio on all Mesos nodes. The remaining steps should be performed from an Alluxio installation
+2. Set the configuration property "alluxio.integration.mesos.alluxio.jar.url" to "LOCAL"
```
Comment (gpang) [inline docs/en/Running-Alluxio-on-Mesos.md]: NIT: use back ticks.

File: docs/en/Running-Alluxio-on-Mesos.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+3. Set the configuration property "alluxio.home" to the path where Alluxio is installed on the Mesos nodes
```
Comment (gpang) [inline docs/en/Running-Alluxio-on-Mesos.md]: NIT: use back ticks.

File: docs/_data/table/cluster-management.csv [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 alluxio.integration.mesos.master.name,AlluxioMaster
 alluxio.integration.mesos.master.node.count,1
-alluxio.integration.mesos.principal,
+alluxio.integration.mesos.principal,alluxio
```
Comment (gpang) [inline docs/_data/table/cluster-management.csv]: what is this changed from empty?

File: docs/_data/table/cluster-management.csv [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
 alluxio.integration.mesos.role,*
 alluxio.integration.mesos.secret,
-alluxio.integration.mesos.user,root
+alluxio.integration.mesos.user,
```
Comment (gpang) [inline docs/_data/table/cluster-management.csv]: why is this changed to unset?

File: docs/_data/table/cluster-management.csv (hunk 1 above)
Comment (aaudiber) [inline docs/_data/table/cluster-management.csv]: This looks much better, thanks

File: docs/_data/table/cluster-management.csv (hunk 4 above)
Comment (aaudiber) [inline docs/_data/table/cluster-management.csv]: It was wrong before, the default has always been "alluxio"

File: docs/_data/table/cluster-management.csv (hunk 5 above)
Comment (aaudiber) [inline docs/_data/table/cluster-management.csv]: setting it as `root` will run the Alluxio Master/Worker as `root`, while setting it to `""` will cause Mesos to determine what's the appropriate account to run as.

File: docs/en/Running-Alluxio-on-Mesos.md (hunk 2 above)
Comment (aaudiber) [inline docs/en/Running-Alluxio-on-Mesos.md]: done

File: docs/en/Running-Alluxio-on-Mesos.md (hunk 3 above)
Comment (aaudiber) [inline docs/en/Running-Alluxio-on-Mesos.md]: done

**Comment by aaudiber** [comment]
@gpang Thanks for the comments, PTAL

File: docs/en/Running-Alluxio-on-Mesos.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
+---
+layout: global
+title: Running Alluxio on Mesos
+nickname: Alluxio on Mesos
+group: Deploying Alluxio
+priority: 4
+---
+
+Alluxio can be deployed through Mesos. This allows Mesos to manage the resources used by Alluxio.
```
Comment (calvinjia) [inline docs/en/Running-Alluxio-on-Mesos.md]: You may want to clarify this. Ie. does it manage JVMs only or also ramdisks, etc.

Comment (aaudiber) [inline docs/en/Running-Alluxio-on-Mesos.md]: Done, and also updated the code to properly calculate how much memory is needed by an Alluxio worker

**Comment by aaudiber** [comment]
@calvinjia Comment addressed, PTAL

**Comment by aaudiber** [comment]
Reopened in branch-1.3: https://github.com/Alluxio/alluxio/pull/3978
